spec/order_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../lib/order'
+
```

Comment:
good thorough tests, however could be better practice regarding the separating of classes and having their own spec files